### PR TITLE
sandbox: Capture timing metrics for API server calls.

### PR DESCRIPTION
**Pull request by SamirTalwar-DA**
`timer` is a superset of `meter`, so this doesn't lose any existing behavior; just adds new behavior.

Part of #5146.

### Changelog

- **[Ledger API Server]** Added timing metrics for all GRPC endpoints.

### Pull Request Checklist

- [x] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [x] Include appropriate tests
- [x] Set a descriptive title and thorough description
- [x] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [x] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: ledger/sandbox/src/main/scala/com/digitalasset/platform/apiserver/MetricsInterceptor.scala [hunk 1]
```diff
@@ -79,12 +81,45 @@ final class MetricsInterceptor(metrics: MetricRegistry) extends ServerIntercepto
...
+  ): ServerCall.Listener[ReqT] = {
     val fullMethodName = call.getMethodDescriptor.getFullMethodName
     val metricName = fullServiceToMetricNameCache.getOrElseUpdate(
       fullMethodName,
       MetricsInterceptor.nameFor(fullMethodName))
-    metrics.meter(metricName).mark()
-    next.startCall(call, headers)
+    val timer = metrics.timer(metricName).time()
+    val listener = next.startCall(call, headers)
+    new TimedListener(listener, timer)
```
Comment (gerolf-da) [inline ledger/sandbox/src/main/scala/com/digitalasset/platform/apiserver/MetricsInterceptor.scala]: How does this behave with streaming responses? Does it only stop the timer when the stream is closed?

Comment (SamirTalwar-DA) [reply ledger/sandbox/src/main/scala/com/digitalasset/platform/apiserver/MetricsInterceptor.scala]: I believe so; it's really hard to tell and I couldn't find a good way to test it, but the numbers look about right.

Comment (SamirTalwar-DA) [reply ledger/sandbox/src/main/scala/com/digitalasset/platform/apiserver/MetricsInterceptor.scala]: OK, wrote some tests; looks good.

File: ledger-api/rs-grpc-akka/BUILD.bazel [hunk 2]
```diff
@@ -29,6 +29,9 @@ da_scala_library(
             "src/test/**/*Test.scala",
         ],
     ),
+    visibility = [
+        "//visibility:public",
```
Comment (stefanobaghino-da) [inline ledger-api/rs-grpc-akka/BUILD.bazel]:
```suggestion
        "//:__subpackages__",
```

**Pull request merged**